Source/WebCore/css/parser/CSSPropertyParserConsumer+Display.cpp

```diff
@@ -162,6 +168,8 @@ RefPtr<CSSValue> consumeDisplay(CSSParserTokenRange& range, CSS::PropertyParserS
             return CSSValueInlineBlock;
         case CSSValueGrid:
             return CSSValueInlineGrid;
+        case CSSValueMasonry:
+            return CSSValueInlineMasonry;
```

Comment:
Do we need to convert `inline masonry` to `inline-masonry` ? We do it for the other ones for compat purposes, but not sure it's valuable for `inline-masonry`. I guess it might needed for the value representation?

Reply:
We do, unless I refactor how we store display values to be a multi-valued struct instead of an enum. :/ I thought about it, but figured that was better to tackle as a follow-up sometime later.